release/ReleaseNotes

```diff
@@ -3,8 +3,29 @@
 
 
 --------------------------------------------------------------------------------------------------------------
-ganga/python/Ganga
+ganga/python/GPIDev
 --------------------------------------------------------------------------------------------------------------
-* ...
+* Fix to MassStorageFile. The local area is now successfully cleaned up after files have been copied
```

Comment:
I think it may be useful (opinions welcome but I was doing it for my release notes) to include the GihHub bug number and/or the PR number. See https://github.com/ganga-devs/ganga/releases/tag/6.1.20 for how this can work.